app/src/main/java/io/github/muntashirakon/AppManager/details/AppDetailsViewModel.java

```diff
@@ -1908,6 +1925,24 @@ private void loadSharedLibraries() {
         mSharedLibraries.postValue(appDetailsItems);
     }
 
+    @NonNull
+    private final MutableLiveData<List<AppDetailsOverlayItem>> mOverlays = new MutableLiveData<>();
+
+    @WorkerThread
+    private void loadOverlays() {
+        if (Build.VERSION.SDK_INT < Build.VERSION_CODES.UPSIDE_DOWN_CAKE
+                || mOverlayManager == null || mPackageName == null)  {
+            mOverlays.postValue(new ArrayList<>());
+            return;
+        }
+        final List<OverlayInfo> overlays =  mOverlayManager.getOverlayInfosForTarget(mPackageName);
+        List<AppDetailsOverlayItem> overlayItems = new ArrayList<>();
+        for (OverlayInfo o : overlays) {
+            overlayItems.add(new AppDetailsOverlayItem(o));
+        }
+        mOverlays.postValue(overlayItems);
```

Comment:
It's possibly throwing an error before reaching here. You can try catching exceptions here for testing.